src/adt/Recoil_Loadable.js

```diff
@@ -32,7 +32,7 @@ type ResolvedLoadablePromiseInfo<+T> = $ReadOnly<{
 
 export type LoadablePromise<T> = Promise<ResolvedLoadablePromiseInfo<T>>;
 
-type Accessors<T> = $ReadOnly<{
+type Accessors<T> = $ReadOnly<{|
```

Comment:
seems like objects are now exact by default, is it not the case in this configuration?

Reply:
I didn't know that, but sure enough I needed to make exact types here manually. Need to check if [utility type](https://flow.org/en/docs/types/utilities/#toc-readonly) behaviours differently